src/Context/SettingsContext.js

```diff
@@ -4,26 +4,26 @@ import { getSettings } from '../apiCalls'
 export const SettingsContext = createContext();
 
 export const SettingsProvider = ({ children }) => {
-    const [ settings, setSettings ] = useState({ workInterval: 12, breakInterval: 7 })
+  const [ settings, setSettings ] = useState({ workInterval: 25, breakInterval: 5 })
 
-    useEffect(() => {
-        getSettings()
-            .then(response => {
-                const workInterval = +response.data.work_interval.split(':')[0]
-                const breakInterval = +response.data.rest_interval.split(':')[0]
-                setSettings({ workInterval, breakInterval })
-            })
-            .catch(err => {
-                if (err.response) {
-                    console.log(err.response.status)
-                }
-            })
-    }, [])
-  
+  useEffect(() => {
+    getSettings()
+      .then(response => {
+        const workInterval = +response.data.work_interval.split(':')[0]
+        const breakInterval = +response.data.rest_interval.split(':')[0]
+        setSettings({ workInterval, breakInterval })
+      })
+      .catch(err => {
+        if (err.response) {
+          console.log(err.response.status)
+        }
+      })
+  }, [])
 
-    return(
-        <SettingsContext.Provider value={[ settings, setSettings ]}>
-            {children}
-        </SettingsContext.Provider>
-    )
+
+  return (
+    <SettingsContext.Provider value={[ settings, setSettings ]}>
+      {children}
+    </SettingsContext.Provider>
+  )
```

Comment:
Not sure why we are getting this weird little bug where the settings are being reset to our default periodically?